src/components/AuthPanel.tsx

```diff
@@ -0,0 +1,143 @@
+
+export const AuthPanel = () => {
+    const [canSeeTest, setCanSeeTest] = useState(false);
+    const [isLogin, setIsLogin] = useState(true);
+
+    const navigate = useNavigate();
+    const ACTION_MESSAGE = isLogin ? "Sign In" : "Sign Up";
+
+    const updateUser = useUserStore((state) => state.updateUser)
+
+    const Copyright = () => {
+        return (
+            <Typography
+                variant="body2"
+                color="text.secondary"
+                align="center"
+                sx={{my: 4}}
+            >
+                {"Copyright © "}
+                <Link color="inherit" href="https://mui.com/">
+                    Cantek NextDoor
+                </Link>{" "}
+                {new Date().getFullYear()}
+                {"."}
+            </Typography>
+        );
+    };
+
+    const handleToggleAuth = () => {
+        setIsLogin((prev) => !prev);
+    };
+
+    // TODO add validation
+    const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        const formData = new FormData(e.currentTarget);
+        const payload = {
+            email: formData.get("email")!.toString(),
+            password: formData.get("password")!.toString(),
+        }
+        const res = isLogin ? await loginUser(payload) : await registerUser(payload);
+        updateUser({...res.data, isLoggedIn: true})
+        navigate("/");
+    };
+
+
+    useEffect(() => {
+        const fetchTest = async () => {
+            try {
+                const res = await jwtTest()
+                console.log("res.data", res.data);
+                setCanSeeTest(true);
+            } catch (e) {
+                console.log("e", e);
+            }
+        };
+
+        fetchTest();
+    }, []);
+
+    return (
+        <div>
+            <CssBaseline/>
+
+            <Box
+                sx={{
+                    marginTop: 8,
+                    display: "flex",
+                    flexDirection: "column",
+                    alignItems: "center",
+                }}
+            >
+                <Avatar sx={{m: 1, bgcolor: "secondary.main"}}>
+                    <LockOutlinedIcon/>
+                </Avatar>
+                <Typography component="h1" variant="h5">
+                    {ACTION_MESSAGE}
+                </Typography>
+                <Box component="form" onSubmit={handleSubmit} noValidate sx={{mt: 1}}>
+
+                    <TextFieldMui autoComplete="email" label="Email Address" name="email" required size="medium"/>
+                    <TextFieldMui autoComplete="current-password" label="Password" name="password" type="password"
+                                  required size="medium"/>
+
+                    {/*<FormControlLabel*/}
+                    {/*  control={<Checkbox value="remember" color="primary" />}*/}
+                    {/*  label="Remember me"*/}
+                    {/*/>*/}
+
+                    <ButtonMui
```

Comment:
Current implementation has no validation, is this intended?

Reply:
Will include in next pr https://github.com/cantek-nextdoor/frontend/issues/3